applications/Web3-deliberative-governance-stack.md

```diff
@@ -0,0 +1,213 @@
+- https://www.linkedin.com/in/remybourganel/
+- https://www.linkedin.com/in/oliviermauco/
+
+## Development Status :open_book:
+
+### Publications 
+
+The first presentation of the problem and path to solution: https://archive.org/details/vergne-metagov
+
+Bourganel, Remy, 2016, Algorithms, diachronic forms, and metadesign, https://www.researchgate.net/publication/305728211_Algorithms_diachronic_forms_and_metadesign 
+
+### references to conversations you might have had related to this project with anyone from the Web3 Foundation,
+
+Discussions with the grant team: David Hawig and Seraya Takahashi, mail conversation with Sebastian Müller
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+- **Total Estimated Duration:** 6 months for milestones 1 and 6 months for milestones 2 / 1 year for milestones 3 and 4 and one year for milestones 5 
+- **Full-Time Equivalent (FTE):**  1,5 FTE for milestones 1 and 2 / 2 FTE for milesteones 3 and 4 and 2 FTE for milestone 5
+- **Total Costs:** The full process of all Milestones demands extended funding. But we do not seek complete funding now. Each milestone can be funded in a consequent order, or partially in combination with other grant-making programs. **The current grant application aims at funding Milestone 1** and if successful milestone 2 which will allow unlocking Milestones 3 to 5. Funding for milestones 3 to 5 is difficult to estimate at this stage. Milestone 1: $25.000 / Milestone 2: $58.000. 
+
+### Milestone 1 - Research paper and Proof of concept (THIS IS THE DEMAND FOR FUNDING)
+
+- **Estimated duration:** 6 months
+- **FTE:**  4 months of a mixed team of junior and senior
+- **Costs:** 25,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | Creative Commons CC-BY |
+| 0b. | Documentation | In this milestone the goal is to publish a proof of concept of the approach, a peer-reviewed article summarising the theoretical framework of the approach, and to produce a detailed roadmap for Milestone 2. We will document the process by the way of a notion site opened to the public. |
+| 0c. | Testing protocol | We will test the approach with key actors of the Web3 / Crypto governance ecosystem in the Dotsama ecosystem and outside (we particularly think about Aave, Paladin, Element.fi, and SNX, who all have interesting governance structures). |
+| 0d. | Docker | We will not provide a docker file as Milestone 1 is research-oriented |
+| 0e. | Article | At the end of Milestone 1 we will publish a peer-reviewed paper consisting of the state of the art of deliberative and crypto governance, a gap analysis, a research and action agenda, and a prototype for experimentation with a real-life DAO or Chain/Parachain. The research will be embedded in the work of the Metagov group (https://metagov.org/) and could be run in collaboration with the Mc Court Institute as both are high-quality networks of researchers on the question. |  
```

Comment:
In my experience, getting a peer-reviewed paper published usually takes at least half a year, assuming it's accepted on first try. Are you sure you want to commit to the article being reviewed _and_ published?

Reply:
Great to hear @DecentralizedScience being active! :)